.github/workflows/deploy-pages-v2.yml

```diff
@@ -3,7 +3,7 @@ name: Deploy v2 to GitHub Pages
 
 on:
   push:
-    branches: ["main"]
+    branches: ["259-siwf-create-assets-directory"]
```

Comment:
@wilwade  it's still not deploying what I'm expecting to the GitHub pages from the index.html. Do you know what's going on there? I was suspecting that it was actually generated but it doesn't look like it.